Source/JavaScriptCore/b3/air/AirFixObviousSpills.cpp

```diff
@@ -577,13 +577,13 @@ class FixObviousSpills {
 
         void sort()
         {
-            std::sort(regConst.begin(), regConst.end(), [] (const RegConst& a, const RegConst& b) {
+            std::ranges::sort(regConst, [] (const RegConst& a, const RegConst& b) {
```

Comment:
Going to assume for the rest, you get the pattern.

```suggestion
            std::ranges::sort(regConst, [](auto& a, auto& b) {
```